doc/release/Communication.md

```diff
@@ -0,0 +1,23 @@
+# Public communications
+
+When releasing a new version a CaliOpen, some public communications have to be done. Follows up a non-exhaustive list:
+
+**The Changelog**
+
+This is visible in release tab on github. It should list the functionalities that have been added, fixed or removed.
+It is a technical content, in english.
+
+**Blog post**
+
+Post an article on https://www.caliopen.org/blog-fr/ in multiple languages. It defines what is the purpose of the release and informations understandable to regular people. It can content a link to github's release page.
+
+**Twitter**
+
+Twitter is a special case due to its constraint and its instantanity.
+Publish a simple short description of the release with a link to the blog post in main available languages.
+The tweet can be reposted 2 or 3 times dependending on language and timezone.
+
+**Mailing list and social networks**
+
+For now it is a simple description of the release with a link to the blog post in main available languages.
+Presently it's necessary to publish on Facebook, Linked In, viadeo, tumblr, VKontakte, renren ...
```

Comment:
Not sure about this media for now as we don't have current presence on them.

Focus on where we exist actually (twitter, mastodon ?), we will open communication to others media later